src/lib/utils.js

```diff
@@ -0,0 +1,71 @@
+import rc from 'rc';
+import fs from 'fs';
+import gitRemoteOriginUrl from 'git-remote-origin-url';
+import getUrlFromGitRemote from './git';
+import loadPropertiesFile from './properties-file';
+
+const gitlabEnvFileName = 'gitlab.env.yml';
+
+async function getConf(cmd, bSingle = false) {
```

Comment:
Having `cmd` here creates tight coupling between this utility method and `program`. To reduce coupling, we could pass `doNotForce` as a boolean argument, or `options` as an object argument, e.g. `{ doNotForce: true} `

Reply:
fixed